### DISCUSSION TITLE
Dicussion about National E1-E6 notices in SDK

### DISCUSSION THREAD

**Discussion by Havunen** [OP-TED/eForms-SDK#789] [Q&A] [answered]
Hi,

We are planning to start implementing national notices using E1-E5 notice types.

Our initial plan is to use BT fields specified in the CELEX_EN_ANNEX for these given notice types. 

In the ANNEX -excel file we can see the list of BT fields used in the given notice type.
If we use EF16 notice type as a basis for E3 national notice type how close it is going to be the official E3 form type?

Are the XML XPaths going to remain the same for the same BT fields in the national notices the same way as they are in EU eforms notices?

For example is the BT-03 form type going to be in the XML in the same location as it is in eF16 form type?
What about the CM fields? Are they going to be similar between EF16 and E3 forms?

**Comment by YvesJo** [comment, marked as answer]
Hi,
A same field shall have a same XPATH independently from the Notice Subtype. 
The set of rules to apply may however vary as the Extended form do not have to follow all the rules commonly applied to other forms. Rules that find their origin in the Business Logic will however have to apply for these forms too.
Be aware that extended notices can't be processed as long as the system is not ready and I invite you to get in touch with the helpdesk for further questions.

**Reply by TateGunning** [reply]
Hello, yes,
We function best inside clear rulesets.
Thank you, will reach out to helpdesk. Happy holidays!
